presto-docs/src/main/sphinx/develop/system-access-control.rst

```diff
@@ -0,0 +1,53 @@
+===================
+SystemAccessControl
```

Comment:
We should use a human readable name "System Access Control" rather than the Java class name